addons/account/demo/account_demo.py

```diff
@@ -108,6 +112,7 @@ def _get_demo_data_move(self, company=False):
                 'invoice_user_id': 'base.user_demo',
                 'invoice_payment_term_id': 'account.account_payment_term_end_following_month',
                 'invoice_date': time.strftime('%Y-%m-01'),
+                'delivery_date': (fields.Date.today() + timedelta(days=-15)).strftime('%Y-%m-%d'),
```

Comment:
```suggestion
                'delivery_date': time.strftime('%Y-%m-01'),
```
Wrong copy/paste? :wink: